panimg/models.py

```diff
@@ -39,6 +43,70 @@ class EyeChoice(str, Enum):
     NOT_APPLICABLE = "NA"
 
 
+class PatientSex(str, Enum):
+    MALE = "M"
+    FEMALE = "F"
+    OTHER = "O"
+
+
+DICOM_VR_TO_VALIDATION_REGEXP = {
+    "AS": re.compile(r"^\d{3}[DWMY]$"),
+    "CS(PatientSex)": re.compile(
+        f"^[{''.join([s.value for s in PatientSex])}]$"
+    ),
+    "DA": re.compile(r"^\d{8}$"),
+    "LO": re.compile(r"^[^\\]{0,64}$"),
+    "PN": re.compile(r"^[^\\]{0,324}$"),
+    "UI": re.compile(r"^[\d.]{0,64}$"),
```

Comment:
I'm not completely sure about the strictness of this validation. It's for value representation `UI` (=StudyInstanceUID, etc). Right now it only allows digits and "." as specified in the DICOM standard. For the ophthalmology usecase a study descriptor like `Visit 6` is required. So the `StudyInstanceUID` field can not be used for that currently. The `StudyDescription` field can be used for it but I'm not sure if that is the right one.

Reply:
We discussed this in the RSE meeting and decided that the StudyDescription is the correct place for this. Any validation that panimg does for these fields should only be according to the fields value representation http://dicom.nema.org/dicom/2013/output/chtml/part05/sect_6.2.html